optimum_benchmark/scenarios/energy_star/scenario.py

```diff
@@ -87,90 +68,84 @@ def __init__(self, config: EnergyStarConfig) -> None:
 
     def run(self, backend: Backend[BackendConfigT]) -> BenchmarkReport:
         if is_torch_distributed_available() and torch.distributed.is_initialized():
-            LOGGER.info("\t+ Distributing batch size across processes")
+            self.logger.info("\t+ Distributing batch size across processes")
             if self.config.input_shapes["batch_size"] % torch.distributed.get_world_size() != 0:
                 raise ValueError(
                     "The batch size must be divisible by the number of processes in a distributed environment"
                 )
             self.config.input_shapes["batch_size"] //= torch.distributed.get_world_size()
```

Comment:
this logic is no longer needed as we do this in the backend now
https://github.com/huggingface/optimum-benchmark/blob/main/optimum_benchmark/backends/pytorch/backend.py#L410-L440
This was introduced to implement TP vs DP logic (same input vs splitting inputs).
You'll have to call these methods after the inputs have been generated and processed, because they might zero the `batch_size` for TP on non-main processes.
https://github.com/huggingface/optimum-benchmark/blob/main/optimum_benchmark/scenarios/inference/scenario.py#L86-L92